src/libstore/build/local-derivation-goal.cc

```diff
@@ -1777,7 +1777,8 @@ void LocalDerivationGoal::runChild()
                     if (pathExists(path))
                         ss.push_back(path);
 
-                dirsInChroot.emplace(settings.caFile, "/etc/ssl/certs/ca-certificates.crt");
+                if (settings.caFile != "")
+                    dirsInChroot.try_emplace("/etc/ssl/certs/ca-certificates.crt", settings.caFile, true);
```

Comment:
(for a follow-up): This syntax is quite unfortunate, we could use designated initializers to make it more understandable

Reply:
Also for later: Maybe we should warn if the file doesn't exist